roles/core/pxe_stack/vars/RedHat_8.yml

```diff
@@ -5,7 +5,7 @@ pxe_stack_apache_user: apache
 pxe_stack_packages_to_install:
   - atftp
   - ipxe-x86_64-bluebanquise
-  #- ipxe-arm64-bluebanquise
+  - ipxe-arm64-bluebanquise
```

Comment:
With this change RedHat_8.yml and CentOS_8.yml are now identical. I would remove CentOS_8.yml to rely on the `os_family` file instead.

Reply:
Ok